app/src/main/java/com/vitorpamplona/amethyst/ui/screen/ChatroomListFeedView.kt

```diff
@@ -99,8 +99,8 @@ private fun FeedLoaded(
     val notificationCache = notificationCacheState.value ?: return
     val context = LocalContext.current.applicationContext
 
-    if (markAsRead.value) {
-        LaunchedEffect(key1 = notificationCache) {
+    LaunchedEffect(key1 = markAsRead.value) {
+        if (markAsRead.value) {
```

Comment:
Swapped the key to use the boolean flag directly for clarity. The behavior appears to be identical, including the number of calls and recompositions, etc.